server/evaluator_test.go

```diff
@@ -91,6 +91,55 @@ func TestEvaluate_FlagNotFound(t *testing.T) {
 	assert.False(t, resp.Match)
 }
 
+func TestEvaluate_FlagNotFoundExcluded(t *testing.T) {
+	var (
+		store = &storeMock{}
+		s     = &Server{
+			logger: logger,
+			store:  store,
+		}
+	)
+
+	disabled := &flipt.Flag{
+		Key:     "bar",
+		Enabled: false,
+	}
+	store.On("GetFlag", mock.Anything, "foo").Return(enabledFlag, nil)
+	store.On("GetFlag", mock.Anything, "bar").Return(disabled, nil)
+	store.On("GetFlag", mock.Anything, "NotFoundFlag").Return(&flipt.Flag{}, errors.ErrNotFoundf(""))
+
+	store.On("GetEvaluationRules", mock.Anything, "foo").Return([]*storage.EvaluationRule{}, nil)
+
+	resp, err := s.BatchEvaluate(context.TODO(), &flipt.BatchEvaluationRequest{
+		RequestId:       "12345",
+		ExcludeNotFound: true,
```

Comment:
would you mind adding another test where `ExcludeNotFound` is false or simply not set (so it defaults to false)?

Reply:
I think that `TestEvaluate_FlagNotFound` is testing this case.

Reply:
I see now that it's testing another grpc call.
I'll add the missing test case

Reply:
done